modules/querying/pages/select-statement/index.adoc

```diff
@@ -0,0 +1,2866 @@
+= SELECT Statement
+:description: Reference documentation for GSQL's SELECT statement.
+:page-aliases: select-statement:readme.adoc, select-statement:README.adoc
+
+NOTE: This page discusses the `SELECT` statement in GSQL Syntax V2, which is the default syntax for TigerGraph version 3.5 and above.
+To learn about the `SELECT` statement in the legacy V1 syntax, see xref:querying:select-statement/select-statement-v1.adoc[]
+
+== Overview
+The `SELECT` statement is an assignment statement with a `SELECT` block on the right-hand side.
+
+The `SELECT` block uses a _path pattern_ to select some of the graph's vertices and edges.
+There are a number of optional clauses that define and/or refine the selection by constraining the vertex or edge set or the result set.
+The final output of a query is either a vertex set known as the _result set_ or a table.
+
+
+There is a maximum size limit of 2 GB for the result set of a `SELECT` block.
+If the result of the `SELECT` block is larger than 2 GB, the system will return no data.
+No error message is produced.
+
+
+.EBNF for GSQL Select Statement
+[source,ebnf]
+----
+gsqlSelectBlock := gsqlSelectClause
+               fromClause
+               [sampleClause]
+               [whereClause]
+               [accumClause]
+               [postAccumClause]*
+               [havingClause]
+               [orderClause]
+               [limitClause]
+
+gsqlSelectClause := vertexSetName "=" SELECT vertexAlias
+----
+
+
+
+The initial clause is the `SELECT` clause: `SELECT vertexAlias`. Its purpose is to specify which set of vertices from the FROM clause is to become the output.
+
+The `SELECT` clause may contain only one item: a vertex alias defined in the FROM clause.
+The vertex alias may be from anywhere in a multi-hop pattern, not only an endpoint.
+
+The <<_from_clause,`FROM` clause>> defines a path pattern to traverse in the graph, and each vertex in the path pattern can be given a vertex alias.
+Thus, the `SELECT` clause picks the set of vertices at one of these points in the pattern -- the source vertices, the target vertices, or those from an interior point in a multi-hop path -- to be the output vertices.
+
+The `SELECT` block has many optional clauses, which fit together in a logical flow.
+Overall, the `SELECT` block starts from a source set of vertices and returns a result set that is either a subset of the source vertices or a subset of their neighboring vertices.
+Along the way, computations can be performed on the selected vertices and edges.
+
+
+[#_from_clause]
+== `FROM` Clause
+
+The `FROM` clause describes either a single hop or a multi-hop path pattern.
+Path patterns also have many other options for finer control and greater flexibility.
+
+.`FROM` clause
+[source,ebnf]
+----
+fromClause := FROM ( stepV2 | pathPattern ["," pathPattern]*)
```

Comment:
What is `stepV2` here?

Reply:
This is from the EBNF in complete EBNF page:

`stepV2 :=  stepVertexSet ["-" "(" stepEdgeSet ")" "-" stepVertexSet]`

It actually looks identical to pathPattern, so I'll just remove it